omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -421,11 +258,6 @@
     end
   end
 
-  # From now on we don't need piptools anymore, uninstall its deps so we don't include them in the final artifact
-  uninstall_buildtime_deps.each do |dep|
-    command "#{python} -m pip uninstall -y #{dep}"
-  end
-
   # Patch applies to only one file: set it explicitly as a target, no need for -p
   if windows_target?
     patch :source => "remove-maxfile-maxpath-psutil.patch", :target => "#{python_3_embedded}/Lib/site-packages/psutil/__init__.py"
```

Comment:
qq: will we one day move the patches to integrations-core? I saw we have a `patches` folder there but it's empty

Reply:
That is the plan yes but I think in this case it's possible for us to remove the patch, I'm waiting on a response: https://github.com/giampaolo/psutil/pull/2110#issuecomment-1282890008